lib/generate_models.rb

```diff
@@ -162,19 +160,19 @@
   { name: 'qdmVersion', type: 'System.String' },
   { name: '_type', type: 'System.String' }
 ]
+datatype_custom_templates = {
+  Patient: 'templates/patient_template.js.erb',
+  Id: 'templates/id_template.js.erb'
+}
 datatypes.each do |datatype, attributes|
-  if datatype == 'Patient'
-    # Handle Patient as its own special case, with its own template.
-    patient_template = File.read('templates/patient_template.js.erb')
-    patient_renderer = ERB.new(patient_template, nil, '-')
-    attrs_with_extras = attributes + extra_fields_js
-    puts '  ' + file_path + datatype + '.js'
-    File.open(file_path + datatype + '.js', 'w') { |file| file.puts patient_renderer.result(binding) }
-  else
-    attrs_with_extras = attributes + extra_fields_js
-    puts '  ' + file_path + datatype + '.js'
-    File.open(file_path + datatype + '.js', 'w') { |file| file.puts renderer.result(binding) }
+  renderer = default_renderer
```

Comment:
This block change seems... awkward. Why not just add another `elsif datatype == 'Id'`?